recipe/meta.yaml

```diff
@@ -2,8 +2,8 @@
 {% set version_major   = "0" %}
 {% set version_minor   = "25" %}
 {% set version_revison = "0" %}
-{% set version = ".".join([version_major, version_minor, version_revison]) %}
-{% set sha256 = "de0ed85fd840dfeb32bcaa94c643307551dc0d967c3714e49087e7edc0cdc571" %}
+{% set version = "0.26.3" %}
+{% set sha256 = "29baa530d6fcbf7cca6a75cf9c78fb88613ca81afb39c62fe492f226f6b61800" %}
```

Comment:
Not sure the right way to remedy this ATM, but maybe you have a better idea than I, @CJ-Wright. 😉

Reply:
Raised as issue ( https://github.com/regro/cf-scripts/issues/48 ).

Reply:
How about we change the recipe to have the version as one string and split into major and minor parts where it is needed. If this is not done in other recipes, no need to tinker with auto-tick?

Reply:
Yeah I think that is what happened `sqlite`. Seems reasonable.

The other option would be to pin the patch release too.

Reply:
I prefer not to pin the patch release.

Reply:
Yeah I vaguely remember us having this conversation before. That's fine.